Makefile

```diff
@@ -1,13 +1,38 @@
 VERBOSE ?= @
 TEST_PROCS ?= $(shell nproc)
+ORIGIN ?= https://github.com/Componolit
+PYTHON_STYLE_HEAD ?= 6440cc638a85a89d486af16eef5541de83e06b54
 
 SHELL := /bin/bash
 PYTEST := python3 -m pytest -n$(TEST_PROCS) -vv --timeout=3600
+CHECKOUT_PYTHON_STYLE := test -d .config/python-style && git -C .config/python-style fetch origin main && git -C .config/python-style -c advice.detachedHead=false checkout $(PYTHON_STYLE_HEAD)
 
 python-packages := bin doc/conf.py examples/apps rflx tests tools stubs setup.py
 
 build-dir := build
 
+.PHONY: init deinit
+
+init: .config/python-style
+	@$(CHECKOUT_PYTHON_STYLE)
+	@rm pyproject.toml
+	@ln -s .config/python-style/pyproject.toml
+	@git update-index --skip-worktree pyproject.toml
+
+deinit:
+	@rm -rf .config/python-style
```

Comment:
Do you see any way to make this a bit less dangerous in case of local modifications? Maybe running some git command that will stop in case of uncommited / unpushed changes?